### PR TITLE
Adding green tea, fruit tea and making tea to black tea.

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
SUMMARY: Content "Adding green tea and fruit tea. Makes tea to black tea."

#### Purpose of change
Adds fruit and green tea into the game. "Tea" is now black tea. Reason for that is that those 3 tea sorts are so different from each other, that they don't belong into the same item.

#### Describe the solution
Adding these teas and spawning them around the world.

#### Describe alternatives you've considered
drinking just tea

#### Testing
Tested in my game. Works as intended.

### CODE REVIEW

File: data/json/items/comestibles/drink.json [hunk 1]
```diff
@@ -1042,6 +1042,30 @@
...
+    "fun": 6
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_fruit",
+    "name": "fruit tea",
+    "copy-from": "tea",
+    "color": "red",
+    "calories": 6,
+    "fatigue_mod": 0,
```
Comment (I-am-Erk) [inline data/json/items/comestibles/drink.json:1064]: These are probably going to produce a "does not update value" error. You don't need to specify an attribute of zero.

File: data/json/items/comestibles/drink.json [hunk 2]
```diff
@@ -1042,6 +1042,30 @@
     "relative": { "fun": 1 },
     "use_action": [  ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_green",
+    "name": "green tea",
+    "copy-from": "tea",
+    "color": "green",
+    "fatigue_mod": 3,
```
Comment (I-am-Erk) [inline data/json/items/comestibles/drink.json:1051]: Green tea is a stronger stimulant than black.

File: data/json/items/comestibles/drink.json [hunk 3]
```diff
@@ -969,7 +969,7 @@
     "symbol": "~",
     "quench": 40,
     "calories": 2,
-    "description": "The beverage of gentlemen everywhere, made from applying hot water to leaves of the tea plant /Camellia sinensis/.",
+    "description": "The beverage of gentlemen everywhere, made from applying hot water to leaves of the tea plant /Camellia sinensis/.  Stronger than green tea.",
```
Comment (I-am-Erk) [inline data/json/items/comestibles/drink.json:972]:
```suggestion
    "description": "The beverage of gentlemen everywhere, made from applying hot water to oxidized leaves of the tea plant /Camellia sinensis/.",
```

I don't think the change here adds anything

File: data/json/items/comestibles/drink.json [hunk 4]
```diff
@@ -1042,6 +1042,30 @@
...
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_green",
+    "name": "green tea",
+    "copy-from": "tea",
+    "color": "green",
+    "fatigue_mod": 3,
+    "stim": 1,
+    "calories": 2,
+    "description": "Made from applying hot water to leaves of the tea plant /Camellia sinensis/, but with a different taste to his brother black tea.",
```
Comment (I-am-Erk) [inline data/json/items/comestibles/drink.json:1054]:
```suggestion
    "description": "Made from applying hot water to leaves of the tea plant /Camellia sinensis/.  Green tea has a lighter, fresher taste than black and is traditionally preferred in Asian cultures.",
```

File: data/json/items/comestibles/drink.json [hunk 5]
```diff
@@ -1042,6 +1042,30 @@
...
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_fruit",
+    "name": "fruit tea",
+    "copy-from": "tea",
+    "color": "red",
+    "calories": 6,
+    "fatigue_mod": 0,
+    "stim": 0,
+    "description": "Tasty fruity tea with a hint of any fruit you can imagine.",
```
Comment (I-am-Erk) [inline data/json/items/comestibles/drink.json:1066]:
```suggestion
    "description": "A tasty beverage made with herbs and dried fruit from plants other than the tea plant. While colloquially called 'tea', technically it's an infusion.",
```

File: data/json/items/comestibles/other.json [hunk 6]
```diff
@@ -755,7 +755,23 @@
...
+    "copy-from": "tea_bag",
+    "description": "Paper sachet with tea leafs inside.  Put it into boiling water to get your cup of green tea."
+  },
+  {
+    "id": "tea_fruit_bag",
+    "type": "COMESTIBLE",
+    "comestible_type": "FOOD",
+    "name": { "str": "fruit tea bag" },
+    "copy-from": "tea_bag",
+    "description": "Paper sachet with tea leafs and fruit parts inside.  Put it into boiling water to get your cup of fruit tea."
```
Comment (I-am-Erk) [inline data/json/items/comestibles/other.json:774]:
```suggestion
    "description": A paper sachet with leaves and fruit parts inside.  Put it into boiling water to make a cup of fruit tea."
```

File: data/json/items/comestibles/other.json [hunk 7]
```diff
@@ -755,7 +755,23 @@
     "material": [ "paper" ],
     "price": 50,
     "price_postapoc": 200,
-    "description": "Paper sachet with tea leafs inside.  Put it into boiling water to get your cup of tea."
+    "description": "Paper sachet with tea leafs inside.  Put it into boiling water to get your cup of black tea."
```
Comment (I-am-Erk) [inline data/json/items/comestibles/other.json:758]:
```suggestion
    "description": "A paper sachet with tea leaves inside.  Put it into boiling water to make a cup of black tea."
```

File: data/json/items/comestibles/other.json [hunk 8]
```diff
@@ -755,7 +755,23 @@
...
+  },
+  {
+    "id": "tea_green_bag",
+    "type": "COMESTIBLE",
+    "comestible_type": "FOOD",
+    "name": { "str": "green tea bag" },
+    "copy-from": "tea_bag",
+    "description": "Paper sachet with tea leafs inside.  Put it into boiling water to get your cup of green tea."
```
Comment (I-am-Erk) [inline data/json/items/comestibles/other.json:766]:
```suggestion
    "description": "A paper sachet with tea leaves inside.  Put it into boiling water to make a cup of green tea."
```

File: data/json/items/comestibles/drink.json (hunk 1 above)
Comment (Lamandus) [reply data/json/items/comestibles/drink.json:1064]: even when the original (tea) has a fatigue mod itself? wouldn't it take the tea value if not added?

File: data/json/items/comestibles/drink.json (hunk 2 above)
Comment (Lamandus) [reply data/json/items/comestibles/drink.json:1051]: Though both contain caffeine, black tea usually has more — making green the better choice for people sensitive to this stimulant. Furthermore, green tea contains more L-theanine, an amino acid that’s calming and can balance the effects of caffeine. 
Source: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4787341/

File: data/json/items/comestibles/drink.json (hunk 1 above)
Comment (I-am-Erk) [reply data/json/items/comestibles/drink.json:1064]: Ah, forgot it was a copy-from, you're probably fine

File: data/json/items/comestibles/drink.json (hunk 2 above)
Comment (I-am-Erk) [reply data/json/items/comestibles/drink.json:1051]: I stand corrected, apparently the teas I used in undergrad were not representative. Nevertheless, the caffeine levels aren't anywhere near as disparate as you have here. I'd suggest at most dropping it to 5 if black tea is 6.

Comment (Lamandus) [reply data/json/items/comestibles/drink.json:1051]: I can live with that. Will change it accordingly.

**Comment by I-am-Erk** [comment]
This is going to need to be run through the [JSON linter](http://dev.narc.ro/cataclysm/format.html) but I am fine with the content

**Comment by Lamandus** [comment]
> 
> 
> This is going to need to be run through the [JSON linter](http://dev.narc.ro/cataclysm/format.html) but I am fine with the content

strange, because I did, but it was wonky for me a few days ago. No Problem, will do again (maybe because I copy and repasted it in notepad++ instead of github itself.

**Comment by anothersimulacrum** [comment]
You need to fix the test errors https://travis-ci.org/github/CleverRaven/Cataclysm-DDA/jobs/699734074#L1134-L1176.

**Comment by Lamandus** [comment]
> 
> 
> You need to fix the test errors https://travis-ci.org/github/CleverRaven/Cataclysm-DDA/jobs/699734074#L1134-L1176.

thanks for pointing that out. I am new to that stuff... but I think now we are fine and good to go.

**Pull request merged**